### PR TITLE
Explicitly save sessionId when starting over

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**
Our new start over (that supported subpaths) wasn't preserving sessionId. This is really the only variable that needs saving upon a start over

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/reducers/create-otp-reducer.js [hunk 1]
```diff
@@ -153,6 +153,10 @@ export function getInitialState(userDefinedConfig) {
     ...(transitModeSettings || [])
   ]
 
+  const sessionId = new URLSearchParams(
```
Comment (binh-dam-ibigroup) [inline lib/reducers/create-otp-reducer.js:156]: Can use `new URLSearchParams(getUrlParams())` from core-utils.

**Review by binh-dam-ibigroup — APPROVED** [review]
Consider using `getUrlParams` function.

File: lib/reducers/create-otp-reducer.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/reducers/create-otp-reducer.js]: Great catch thanks!

**Review by josh-willis-arcadis — APPROVED** [review]
LGTM

**Pull request merged**